Build/Build.cs

```diff
@@ -261,6 +261,7 @@ class Build : NukeBuild
     Target Push => _ => _
         .DependsOn(Pack)
         .OnlyWhenDynamic(() => IsTag)
+        .ProceedAfterFailure()
```

Comment:
❓ Since it's the last target in the chain, why do we need this one?

Reply:
As I understand the `.DependsOn()`: This works in opposite direction, meaning: we start at `Push` (in the main execute method) and then backwards until we reach the end (in this case `Clean`).

And because `Push` is the first "entry point" of the build pipeline, we need this at that place

Reply:
If `public static int Main() => Execute<Build>(x => x.SpellCheck, x => x.Push);` runs the targets consecutively and in the specified order, we strictly don't need `ProceedAfterFailure` here as `Push` is the last target.
But I'm fine having it here.

Reply:
cool